scripts/run_locally_with_docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu
+
+DOCKER_IMAGE_NAME=document-download-api
+
+docker run -it --rm \
+  -e NOTIFY_ENVIRONMENT=$NOTIFY_ENVIRONMENT \
+  -e FLASK_APP=$FLASK_APP \
+  -e FLASK_DEBUG=$FLASK_DEBUG \
+  -p 7000:7000 \
+  -v $(pwd):/home/vcap/app \
+  ${DOCKER_IMAGE_NAME} \
+  flask run --host 0.0.0.0 -p 7000
```

Comment:
Might be nice to define/read the port from an env var and use that in all three places here.

Reply:
Good suggestion, will implement and squash in